### PR TITLE
IT-4618 | update radius, add some paddings for balloon and add fixed full width to input

### PR DESCRIPTION
**Pull request by tweetgeek**

### CODE REVIEW

File: src/ui/mathliveview.js [hunk 1]
```diff
@@ -13,6 +13,9 @@ export default class MathLiveView extends InputView {
 				input: [
 					bind.to( () => ( this.value = this.element.getValue( 'latex-expanded' ) ) ),
 					bind.to( 'input' )
+				],
+				mount: [
+					bind.to( () => this.element.setOptions( this._options ) )
```
Comment (kvas-damian) [inline src/ui/mathliveview.js:18]: do we still need `render` method?

File: theme/mathform.css [hunk 2]
```diff
@@ -23,6 +27,16 @@
 			flex-basis: 50%;
 		}
 	}
+
+	& .ck-math-view {
+		padding: 16px;
+
+		& .ck-labeled-input {
```
Comment (kvas-damian) [inline theme/mathform.css]: is this selector necessary?

File: theme/mathform.css [hunk 3]
```diff
@@ -1,3 +1,7 @@
+.ck.ck-rounded-corners > .ck-balloon-panel {
+	border-radius: 4px;
```
Comment (kvas-damian) [inline theme/mathform.css]: maybe we should override `--ck-border-radius` for entire CKEditor?

File: src/ui/mathliveview.js (hunk 1 above)
Comment (tweetgeek) [reply src/ui/mathliveview.js:18]: is not needed, deleted

File: theme/mathform.css (hunk 3 above)
Comment (tweetgeek) [reply theme/mathform.css]: i ovverride `--ck-border-radius`, a lot has not changed in the editor, but balloons look better and consistent

File: src/mathui.js [hunk 4]
```diff
@@ -96,6 +97,10 @@ export default class MathUI extends Plugin {
 			this._closeFormView();
 		} );
 
+		this.listenTo( formView, 'mathlive:virtualKeyboard:toggle', ( event, state ) => {
```
Comment (pioziol123) [inline src/mathui.js:100]: look at https://github.com/bethinkpl/ckeditor5-bethink/blob/main/src/plugin-bethink-lucidcharts.js#L21
maybe we can delegate events in the same way but into the editor directly

Comment (tweetgeek) [reply src/mathui.js:100]: I tried this, but I would have to pass an instance of the editor or pluins down to MathLiveView and it ended up with circural dependencies error

**Pull request merged**